hw2/01-coins.js

```diff
@@ -1,7 +1,40 @@
 /** Exercise 01 - Coins **/
 
 const calculateChange = (input) => {
-  // Add your code here
+  var amount = input;
+  var output = "$" + input + " ==> ";
+  if (input < 0) {
+    output += " Error: the number is too small";
```

Comment:
The style guide suggests Using single quotes instead of double quotes for strings.